src/components/application_manager/include/application_manager/application.h

```diff
@@ -786,6 +786,31 @@ class Application : public virtual InitialApplicationData,
    */
   virtual uint32_t GetAvailableDiskSpace() = 0;
 
+#ifdef SDL_REMOTE_CONTROL
+  virtual void set_system_context(
```

Comment:
please add descriptions of the new added functions and parameters

Reply:
Confirmed Resolved 👍